content/blog/2024/02/camel-whatsapp-langchain4j/index.md

```diff
@@ -0,0 +1,148 @@
+---
+title: "Integrate your AI models effortlessly with Apache Camel"
+date: 2024-02-09
+draft: true
+authors: [croway]
+categories: ["Camel Spring Boot", "Usecases", "AI"]
+preview: "Build a conversational AI integration with Apache Camel, langchain4j, and WhatsApp."
+---
+
+
+This blog shows how Apache Camel can help integrate multiple systems with an AI model, in particular, the [camel-whatsapp](/components/4.0.x/whatsapp-component.html) component is used to build a chat on WhatsApp; so that a user can easily communicate with the LLM via WhatsApp.
```

Comment:
I think you need to define the acronym LLM as first occurence (or provide a link)